Dockerfile

```diff
@@ -0,0 +1,48 @@
+FROM condaforge/mambaforge:latest
+
+# For opencontainers label definitions, see:
+#    https://github.com/opencontainers/image-spec/blob/master/annotations.md
+LABEL org.opencontainers.image.title="RAiDER"
+LABEL org.opencontainers.image.description="Raytracing Atmospheric Delay Estimation for RADAR"
+LABEL org.opencontainers.image.authors="David Bekaert<david.bekaert@jpl.nasa.gov>, Jeremy Maurer <maurer.jeremy@gmail.com>, Piyush Agram <piyush@descarteslabs.com>"
```

Comment:
@jhkennedy I suggest to add @sssangha to this list as the primary developer of the GNSS interface and stats modules